pkg/plugins/backendplugin/manager.go

```diff
@@ -222,11 +235,13 @@ func (m *manager) CallResource(config PluginConfig, reqCtx *models.ReqContext, p
 
 	clonedReq := reqCtx.Req.Clone(reqCtx.Req.Context())
 	keepCookieNames := []string{}
-	if config.JSONData != nil {
-		if keepCookies := config.JSONData.Get("keepCookies"); keepCookies != nil {
-			keepCookieNames = keepCookies.MustStringArray()
-		}
-	}
+	/// TODO! THIS SEEMS IMPORTANT BUT I DO NOT GET IT
```

Comment:
Don't skip this.

Reply:
You can configure this in data source config if you're using the HTTPSettings react component in your data source plugin config editor. Should check `DataSourceInstanceSettings` and proceed if not nil.